apps/studio/components/interfaces/Auth/Hooks/HookCard.tsx

```diff
@@ -0,0 +1,108 @@
+import { Check, Webhook } from 'lucide-react'
+import { Badge, Button, Input } from 'ui'
+import { Hook } from './hooks.constants'
+
+interface HookCardProps {
+  hook: Hook
+  canUpdateConfig: boolean
+  onToggle: (enabled: boolean) => void
+  onSelect: () => void
+}
+
+export const HookCard = ({ hook, canUpdateConfig, onToggle, onSelect }: HookCardProps) => {
+  return (
+    <div className="bg-surface-100 border-default overflow-hidden border shadow px-5 py-4 flex flex-row">
+      <div className="px-3 pt-1">
+        <Webhook size={24} strokeWidth="0.5" />
+      </div>
+      <div className="flex flex-col flex-0 overflow-y-auto w-full">
+        <span className="text-sm text-foreground">{hook.title}</span>
+        <span className="text-sm text-foreground-lighter">{hook.subtitle}</span>
+        <div className="text-sm flex flex-row space-x-5 py-4">
+          {hook.method.type === 'postgres' ? (
+            <div className="flex flex-col w-full space-y-2">
+              <div className="flex flex-row items-center">
+                <span className="text-foreground-light w-20">type</span>
+                <span className="text-foreground">Postgres function</span>
+              </div>
+              <div className="flex flex-row items-center">
+                <span className="text-foreground-light w-20">schema</span>
+                <Input
```

Comment:
we need new components for these
![Uploading Screenshot 2024-05-21 at 11.00.23 AM.png…]()

Reply:
You probably had an old branch, this is how they look now.

![Screenshot 2024-05-27 at 13 53 50](https://github.com/supabase/supabase/assets/568291/f7e1835d-5cc8-4d3b-a616-7480e7c41d2b)